projects/packages/my-jetpack/_inc/constants.js

```diff
@@ -1,2 +1,3 @@
 export const MY_JETPACK_MY_PLANS_MANAGE_SOURCE = 'my-jetpack-my-plans-manage';
+export const MY_JETPACK_MY_PLANS_PURCHASE_SOURCE = 'jetpack-plans';
```

Comment:
Even though `jetpack-plans` source already exists and takes us to the place we want, the idea of redirect sources is that they are unique to a `source`, or to a place where the link exists.

This allows us to change the destination of a single link, and not the destination of all the links that point to the same place (if we wanted to do that we could simply add a redirect on our server, since we control the domain).

I suggest `my-jetpack-my-plans-purchase`

Reply:
Thanks, addressed in d47dcca75dc5cd0157b031a3e9fbfa0888f004f3